Update PR #1061 with the changes discussed in the FuSa call to make t… (#1141)
spdx/spdx-3-model: model/Core/Classes/Requirement.md

SPDX-License-Identifier: Community-Spec-1.0

# Requirement

## Summary

> A distinct unit representing a requirement as it is used by systems, software and hardware engineering.

Reviewer: ```suggestion
A distinct unit representing a requirement, as used in systems, software, and hardware engineering.
```

## Description

> A requirement element is a distinct article or unit defining an expectation, need, behaviour, design intent etc. of an item that already exists or is to be created based on this requirement.

Reviewer: ```suggestion
A requirement element is a distinct unit that defines an expectation, need, behavior, or design intent of an item that either already exists or is to be created in accordance with this requirement.
```

> A requirement element is a distinct article or unit defining an expectation, need, behaviour, design intent etc. of an item that already exists or is to be created based on this requirement.

Reviewer: ```suggestion
A requirement element is a distinct article or unit defining an expectation, need, behaviour, design intent, etc., of an item that already exists or is to be created based on this requirement.
```
